src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -59,13 +59,6 @@
          used to build the project/solution.
   -->
 
-  <PropertyGroup>
```

Comment:
There's some text about 2019 above, what do we want to do with it?

Reply:
I want to keep this info because it would be still relevant when we move to a newer version of VS we change the version then.